firmware_configurations/klipper/RealDeuce/MKS-Makerbase/Monster8_v1.0_003/Voron2_Monster8_Config.cfg

```diff
@@ -0,0 +1,531 @@
+## if your fan is not able to slow down effectively
+off_below: 0.10
+
+[heater_fan controller_fan]
+#! In HE1 Positon (Controller fan)
+#pin: PB0
+#kick_start_time: 0.5
+heater: heater_bed
+heater_temp: 45.0
+
+# Uncomment this for a GCode controllable fan
+#[heater_fan exhaust_fan]
+#! FAN2 Connector (Exhaust fan)
+pin: PA0
+#max_power: 1.0
+#shutdown_speed: 0.0
+##kick_start_time: 0.5
+#heater: heater_bed
+#heater_temp: 60
+#fan_speed: 1.0
+
+# Uncomment this for an automatic chamber temperature fan
+#[temperature_fan chamber]
+##! FAN2 Connector (Exhaust fan)
+#pin: PA0
+#max_power: 1.0
+#shutdown_speed: 0.0
+#sensor_type: NTC 100K beta 3950
+##! In TH1 Position (Chamber Temperature)
+#sensor_pin: PC2
+#control: watermark
+#min_temp: 0
+#max_temp: 70
+#target_temp: 35.0
+#gcode_id: C
+
+#####################################################################
+#   LED Control
+#####################################################################
+
+#[output_pin caselight ]
+#! In HE2 Position (Chamber Lighting)
+#pin: PA3
+#pwm: true
+#shutdown_value: 0
+#cycle_time: 0.01
+
+#####################################################################
+#   Homing and Gantry Adjustment Routines
+#####################################################################
+
+[idle_timeout]
+timeout: 1800
+
+[safe_z_home]
+## XY Location of the Z Endstop Switch
+## Update -10,-10 to the XY coordinates of your endstop pin
+## (such as 157,305) after going through Z Endstop Pin
+## Location Definition step.
+home_xy_position:-10,-10
+speed:100
+z_hop:10
+
+[quad_gantry_level]
+## Use QUAD_GANTRY_LEVEL to level a gantry.
+## Min & Max gantry corners - measure from nozzle at MIN (0,0) and
+## MAX (250, 250), (300,300), or (350,350) depending on your printer size
+## to respective belt positions
+
+#--------------------------------------------------------------------
+##      Gantry Corners for 250mm Build
+##      Uncomment for 250mm build
+#gantry_corners:
+#       -60,-10
+#       310, 320
+##      Probe points
+#points:
+#       50,25
+#       50,175
+#       200,175
+#       200,25
+
+##      Gantry Corners for 300mm Build
+##      Uncomment for 300mm build
+#gantry_corners:
+#       -60,-10
+#       360,370
+##      Probe points
+#points:
+#       50,25
+#       50,225
+#       250,225
+#       250,25
+
+##      Gantry Corners for 350mm Build
+##      Uncomment for 350mm build
+#gantry_corners:
+#       -60,-10
+#       410,420
+##      Probe points
+#points:
+#       50,25
+#       50,275
+#       300,275
+#       300,25
+
+#--------------------------------------------------------------------
+speed: 100
+horizontal_move_z: 10
+retries: 5
+retry_tolerance: 0.0075
+max_adjust: 10
+
+#####################################################################
+#   Displays
+#####################################################################
+
+#--------------------------------------------------------------------
+
+[display]
```

Comment:
I would comment out the entire `Displays` section, since you normally get the display up and running after you get klipper configured and running.

Reply:
It seems to be enabled in all the other example configurations... and I've never treated the display as a separate step from Klipper.

Reply:
Looks like it's uncommented in the spider config, but commented out in the SKR1.3/1.4 configs, so who knows.